### ISSUE TITLE
UI show show epoch counts (not just iteration counts)

### ISSUE THREAD

**Issue by DonaldAlan** [deeplearning4j/deeplearning4j#5448]
http://localhost:9000/train/overview shows the iteration count. But it doesn't show the epoch count. It should!

**Comment by DonaldAlan** [comment]
It might be good to show in the graphs a vertical line each time a new epoch starts.  If the training data has weird ordering, the learning rate may vary depending on where in the training data you are. If you know when the epoch started, you'd see the periodicity.

**Comment by DonaldAlan** [comment]
Another question/idea. After training finishes (or I kill it), the UI doesn't respond (of course). It would be cool if the data or graphs were saved on disk so we could view it later. Alternatively, start a server to track such things.

**Comment by agibsonccc** [comment]
Closing — this issue is over 7 years old with no recent activity. The codebase has undergone a major rewrite. If this is still relevant, please open a new issue with current reproduction steps.
